### PR TITLE
Update workflow-run-with-alert-modal.tsx

### PR DESCRIPTION
**Pull request by ydv129**
Changes and Additions
1) Fixed Syntax Error: Corrected the className attribute for the dynamic fields. 2) Fetch Workflows: Added a useEffect hook to fetch workflows from the specified GitHub URL. 3) Error Handling: Implemented error handling to display an error message if the workflows cannot be fetched. 4) Fetched Workflows Display: Added a section to display the fetched workflows.

Closes # 

## ✅ Checks

- [x] My pull request adheres to the code style of this project
- [x] My code requires changes to the documentation
- [x] I have updated the documentation as required
- [x] All the tests have passed

### CODE REVIEW

File: keep-ui/app/workflows/workflow-run-with-alert-modal.tsx [hunk 1]
```diff
@@ -38,6 +38,29 @@ export default function AlertTriggerModal({
...
+        const data = await response.json();
+        setWorkflows(data);
+        setFetchError(false);
+      } catch (error) {
+        setFetchError(true);
+      }
+    };
+
+    fetchWorkflows();
+  }, []);
```
Comment (talboren) [inline keep-ui/app/workflows/workflow-run-with-alert-modal.tsx:63]: worth introducing a new hook for this, wdyt?

Comment (ydv129) [reply keep-ui/app/workflows/workflow-run-with-alert-modal.tsx:63]: Crafting a captivating hook is essential to engage the audience from the outset. A well-designed hook can pique the interest of the listeners, setting the tone for a successful presentation. In this context, introducing a new hook can be a valuable strategy to revitalize the presentation and make it more compelling

**Review by talboren — CHANGES_REQUESTED** [review]
I’m sorry but this is not at all what the issue describes.

File: keep-ui/app/workflows/workflow-run-with-alert-modal.tsx (hunk 1 above)
Comment (talboren) [reply keep-ui/app/workflows/workflow-run-with-alert-modal.tsx:63]: Huh?

**Comment by talboren** [comment]
I’m going to move further and close the PR as this seems to be some automatic bot. Please reopen it if I’ve misunderstood the situation.

**Pull request closed without merging**